Test.Shared/WebSocketMock.swift

```diff
@@ -31,6 +31,10 @@ class WebSocketMock: WebSocket {
         return mockConnected
     }
 
+    convenience init() {
+        self.init(url: URL(string: "http://doesnt.matter")!)
```

Comment:
Can you don't use `!`? Let's make sure the URL exists otherwise we fail the test.